src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/StreamingTests.cs

```diff
@@ -17,15 +17,30 @@
 
 public class StreamingTests : ConditionalWcfTest
 {
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+    [ActiveIssue(832)] // Windows Stream Security is not supported in NET Native
```

Comment:
Non-blocking observation only: as pointed out in a separate pre-PR code review, this is a good example of an ActiveIssue that needs to be retained when we ultimately move to Helix.  It is a true product issue on NET Native and unrelated to ToF or Xunit.  We agreed that temporarily FULLXUNIT_NOTSUPPORTED was an acceptable compromise but need to be aware to manually migrate these #if'ed out ActiveIssues ourside the #if or they will fail in Helix.